tests/rest/media/v1/test_url_preview.py

```diff
@@ -140,9 +145,9 @@ def resolveHostName(
                 resolutionReceiver.resolutionComplete()
                 return resolutionReceiver
 
-        self.reactor.nameResolver = Resolver()
+        self.reactor.nameResolver = Resolver()  # type: ignore[assignment]
```

Comment:
Rather annoying that `MemoryReactor` doesn't implement [`IReactorPluggableResolver`](https://twistedmatrix.com/documents/current/api/twisted.internet.interfaces.IReactorPluggableResolver.html). 👍